internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -365,6 +370,13 @@ func (r *ClusterDeploymentsReconciler) installDay1(ctx context.Context, log logr
 			return ctrl.Result{Requeue: true, RequeueAfter: longerRequeueAfterOnError}, nil
 		}
 
+		// Ensure release image exists in versions cache
+		_, err = r.addReleaseImage(ctx, clusterInstall.Spec, pullSecret)
+		if err != nil {
+			log.WithError(err).Error("failed to add OCP version")
```

Comment:
this line is repeated both here and in `installDay2Hosts()`, maybe it should be included in `addReleaseImage()`?
Additionally, this is not informative enough to the reader. please also print for which image name this is failing.

Reply:
sure, added.